elytron-common/src/main/java/org/wildfly/extension/elytron/KeyStoreDefinition.java

```diff
@@ -226,16 +238,36 @@ protected void performRuntime(ModelNode result, ModelNode operation, KeyStoreSer
     @Override
     public void registerOperations(ManagementResourceRegistration resourceRegistration) {
         super.registerOperations(resourceRegistration);
-        resourceRegistration.registerOperationHandler(LOAD, PersistanceHandler.INSTANCE);
+        resourceRegistration.registerOperationHandler(loadOperation, PersistanceHandler.INSTANCE);
         if (isServerOrHostController(resourceRegistration)) {
-            resourceRegistration.registerOperationHandler(STORE, PersistanceHandler.INSTANCE);
+            resourceRegistration.registerOperationHandler(storeOperation, PersistanceHandler.INSTANCE);
+        }
+    }
+
+    private static class KeyStoreOperations {
```

Comment:
Why is this class needed?

Reply:
I think I added it at some point to deal with static method issues I was running into. It's not needed anymore so I'll remove it